### PR TITLE
Minor phpdoc and PSR-2 fixes

### PR DESCRIPTION
**Pull request by bcremer**
While digging deeper in the code my IDE noticed some minor phpdoc and PSR-2 issues.

### CODE REVIEW

File: src/Block/Renderer/DocumentRenderer.php [hunk 1]
```diff
@@ -21,9 +21,9 @@
 class DocumentRenderer implements BlockRendererInterface
 {
     /**
-     * @param Document $block
+     * @param AbstractBlock|Document $block
+     * @param HtmlRendererInterface $htmlRenderer
      * @param bool $inTightList
-     *
```
Comment (colinodell) [inline src/Block/Renderer/DocumentRenderer.php]: Could we keep this empty line between the `param` and `return`?  I like using the PEAR standard of having exactly one empty line in between.

Comment (colinodell) [inline src/Block/Renderer/DocumentRenderer.php]: Actually nvm, I shouldn't be asking you to make more changes simply to please my inner OCD, especially when your improvements are great as-is.

**Comment by colinodell** [comment]
Thanks for the contribution @bcremer!

File: src/Block/Renderer/DocumentRenderer.php (hunk 1 above)
Comment (bcremer) [inline src/Block/Renderer/DocumentRenderer.php]: Thanks for the merge. I can totally relate to your inner OCD in regards to coding styles ;)

**Pull request merged**